json/src/test/java/org/apache/JsonGenericSerializerTest.java

```diff
@@ -78,6 +80,7 @@ void serializeShouldHandleAllKnownTypes(BaseType domainObject, String serialized
     @ParameterizedTest
     @MethodSource
     void deserializeShouldHandleAllKnownTypes(BaseType domainObject, String serializedJson) throws Exception {
+        @SuppressWarnings({ "rawtypes", "unchecked" })
```

Comment:
why this one is not treated like the others?

Reply:
fixed, sorry